Makefile

```diff
@@ -6,6 +6,10 @@ ASFLAGS = --32
 LDFLAGS = $(CFLAGS) -B. -Wl,-b,coff-i386 -no-pie
 LDLIBS = -lncurses -ltinfo
 PATH := .:$(PATH)
+prefix ?= /
+bindir = ${prefix}usr/local/bin
+sharedir = ${prefix}usr/local/share/lotus
+profiledir = ${prefix}etc/profile.d
```

Comment:
I'd really appreciate if you set prefix to /usr/local and set bindir and sharedir to ${prefix}/bin and ${prefix}/share

That way package maintainers can `make prefix=$packageroot/usr/ install` or similar